changes-set.txt

```diff
@@ -26,6 +26,18 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+ 2-Oct-23 xpsfeqcda xpsfeq      changes from +c to a pair of ordered pairs
+ 2-Oct-23 xpscfv    fvprif      changes from +c to a pair of ordered pairs
```

Comment:
Why do the comments of the two functions differ in detail, and why is fvprif valid for a domain of just one element?

Reply:
The comment of `fvprif` was incorrect - there are always two elements there because of the 𝐴 ∈ 𝑉 ∧ 𝐵 ∈ 𝑊 condition. I've pushed up a fix to use the comment which had been on `xpsfeq`